src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -307,7 +308,7 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
     valueCaption: ComponentOptions.buildJsonOption<IStringMap<string>>({ defaultValue: {} }),
 
     /**
-     * The [id](@link Facet.options.id) of another facet in which at least one value must be selected in order
+     * The id of another facet in which at least one value must be selected in order
```

Comment:
Linking only the facet component made no sense I think @fbeaudoincoveo

Reply:
Ah, indeed... Maybe we could say "The `id` option value"?

Just so people don't confuse this with the id HTML attribute... what do you think?